docs/api/domains.md

```diff
@@ -135,5 +135,25 @@ const Planets = {
   }
 }
 
-repo.push(Actions.add, { name: 'earth' }) // this will add Earth
+repo.push(addPlanet, { name: 'earth' }) // this will add Earth
+```
+
+Registration is given the command associated with an action, and the
```

Comment:
Good question. We have some terms issues. We call the functions you pass to `repo.push` an action, or at least, we store them under `./actions`, but we also call the thing returned from `repo.push` an action.

So I've started calling the things you pass to `repo.push` _commands, like:

```
repo.push(command, paramOne, paramTwo)
```

What do you think?